homework/week4/chrispark0304/15649.c

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+void print_arr(int arr[], int len){
+    for(int i = 0 ; i < len ; i++){
+        if(i == len - 1){
+            printf("%d\n", arr[i]);
+        }
+        else{
+            printf("%d ",arr[i]);
+        }
+    }
+}
+
+void permutation(int arr_isused[], int size_arr_isused, int printed_arr[], int size_printed_arr, int depth, int curr_depth){
+    if(depth == curr_depth){
+        for(int i = 1 ; i < size_arr_isused ; i++){
+            if(arr_isused[i] != 0) continue;
```

Comment:
한줄이여도 개행 및 중괄호 해주세요.
```suggestion
            if(arr_isused[i] != 0) {
                continue;
            }
```